app/models/micropost.rb

```diff
@@ -0,0 +1,17 @@
+class Micropost < ApplicationRecord
+  mount_uploader :picture, PictureUploader
+
+  belongs_to :user
+
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
```
2 cách vaildate này giống và khác gì nhau?

Reply:
Theo ý kiến của em, việc validate presence user_id chỉ chắc chắn rằng user_id sẽ ko nil hoặc rỗng. Nó có thể là invalid id, hoặc tệ hơn, nó có thể trỏ vào user_id đã xóa rồi.
Còn validate presence user chắc chắn rằng nó sẽ tham chiếu tới một object tồn tại (bằng code bằng thịt)